src/main/java/de/terrestris/momo/service/GeoServerFontService.java

```diff
@@ -0,0 +1,86 @@
+package de.terrestris.momo.service;
+
+import java.net.URISyntaxException;
+
+import org.apache.http.Header;
+import org.apache.http.HttpException;
+import org.apache.http.message.BasicHeader;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import de.terrestris.shogun2.util.http.HttpUtil;
+import de.terrestris.shogun2.util.model.Response;
+
+/**
+ *
+ * @author Johannes Weskamm
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+@Service("geoServerFontService")
+public class GeoServerFontService {
+
+	/**
+	 */
+	private static final Logger LOG = Logger.getLogger(GeoServerFontService.class);
+
+	/**
+	 *
+	 */
+	@Value("${geoserver.baseUrl}")
+	private String geoServerBaseUrl;
+
+	/**
+	 *
+	 */
+	@Value("${geoserver.username}")
+	private String gsuser;
+
+	/**
+	 *
+	 */
+	@Value("${geoserver.password}")
+	private String gspassword;
+
+	/**
+	 * Get the list of available fonts in GeoServer
+	 * @return
+	 * @throws HttpException
+	 * @throws URISyntaxException
+	 */
+	public Response getGeoServerFontList() throws URISyntaxException, HttpException{
+		String url = geoServerBaseUrl.split("/momo/ows")[0] + "/rest/resource/fonts";
```

Comment:
Non-blocking here, but I would suggest to have the webapp base part (`momo`) of the URL configurable. Is this change relevant for the `shogun2-webapp`?

Reply:
Next playday, this is currently a project solution

Reply:
I'll remind you!